sdk/communication/azure-communication-callautomation/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.0-beta.2 (2024-09-06)
```

Comment:
I think we have released from callautomation/release/beta-1  branch let me double check on this

Reply:
Yes we have released from beta-1 https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4030250&view=results, please can you rebase from beta-1?

Reply:
![image](https://github.com/user-attachments/assets/84bd588f-d2f3-49d8-bd52-e6fe62e0ff95)

Reply:
Fixed it